addon_service/user_reference/models.py

```diff
@@ -16,6 +17,12 @@ def configured_storage_addons(self):
             base_account__account_owner=self,
         )
 
+    @property
+    def configured_resources(self):
+        return ResourceReference.objects.filter(
+            configured_storage_addons__base_account__account_owner=self,
```

Comment:
will this get duplicates if one resource has multiple configured addons by the same user? If so, might need to be something like:
```
ResourceReference.objects.annotate(
  has_addon_configured_by_user=Exists(
    ConfiguredStorageAddon.objects.all.filter(
      confiugred_resource_id=OuterRef('id'),
      base_account__account_owner=self
    )
).filter(has_addons_configured_by_user=True)
```

Reply:
oh, probably yes duplicates -- how do you think the subquery compares to thowing a `.distinct()` on there?

Reply:
My understanding is that `distinct` performance is usually pretty poor.

Reply:
feels like a case (with probably usually low row counts and multiplicity) where `SELECT DISTINCT` would do fine enough, but haven't benchmarked and am fine with the subquery